docs/standard/containerized-lifecycle-architecture/Docker-application-lifecycle/containers-foundation-for-devops-collaboration.md

```diff
@@ -62,5 +62,5 @@ Here are some of the most important benefits provided by a solid DevOps workflow
 -   Plug and play well with many of your existing DevOps investments, including investments in open source
 
 >[!div class="step-by-step"]
-[Previous] (index.md)
-[Next] (../Microsoft-platform-tools-containerized-apps/index.md)
+[Previous](index.md)
+[Next](../Microsoft-platform-tools-containerized-apps/index.md)
```

Comment:
@v-alje that looks that real issues to fix are in the following folder:
https://github.com/dotnet/docs/tree/master/docs/standard/containerized-lifecycle-architecture

My recommendation to you is to create a new PR that updates the files only from that folder: that one would be much easier to review and most probably it would be the one catching the majority of such mistakes in the docs.